framework/components/ecl-blockquotes/_blockquotes-mixins.scss

```diff
@@ -0,0 +1,22 @@
+@mixin blockquotes() {
+  border-left: 0.417em solid $ecl-color-secondary;
```

Comment:
`0.417em`... maybe it's time to refactor this. I think it's okay to give it an absolute value in pixels (still better than `0.417em`)